Logic/igtlioConnector.cxx

```diff
@@ -689,7 +689,7 @@ CommandDevicePointer Connector::SendCommand(std::string device_id, std::string c
   vtkSmartPointer<CommandDevice> device = CommandDevice::SafeDownCast( AddDeviceIfNotPresent(key) );
 
   igtlio::CommandConverter::ContentData contentdata = device->GetContent();
-  contentdata.id +=1;
```

Comment:
This would seem to imply that the last used command ID was in the ContentData of the device. Is a new device created for each command?

Reply:
Ah, you're right. I was setting a new id for each command, which created a new device.
Nevermind, ignore this.